Tests/Find-GlobFile.Tests.ps1

```diff
@@ -291,4 +298,21 @@ else
             ThenFound '.file','.hidden\file2.txt'
         }
     }
+}
+
+if( $IsWindows )
+{
+    Describe 'Find-GlobFile.when excluding a directory' {
+        It 'should not recurse into excluded directories' {
+            Init
+            GivenFile 'file.txt','excluded\file.txt'
+            # Create a circular junction. When you recurse too deeply, eventually PowerShell throws an error that you've
+            # gone too deep. If it got excluded properly, there won't be an error.
+            New-CJunction -Link (Join-Path -Path $testRoot -ChildPath 'excluded\circular') `
+                          -Target (Join-Path -Path $testRoot -ChildPath 'excluded')

```

Comment:
Does this still apply if a system has [NTFS long paths](https://docs.microsoft.com/en-us/windows/win32/fileio/naming-a-file#enable-long-paths-in-windows-10-version-1607-and-later) enabled?

Reply:
🤷‍♀  If they do, then the test will just run and run and run and run. Maybe eventually Windows will run out of memory and crash?